locarise_drf_oauth2_support/oauth2_client/views.py

```diff
@@ -16,6 +17,13 @@
 User = get_user_model()
 
 
+def get_data(request):
```

Comment:
According to the `setup.py` file, the goal of this lib is not to support DRF2 but only DRF3.